director/models/__init__.py

```diff
@@ -42,6 +42,13 @@ def save(self):
             db.session.rollback()
             raise
 
+    def commit(self):
```

Comment:
Maybe it's possible to reuse this `commit` function into the `save` one, like this :

```
def commit(self):
    try:
        db.session.commit()
    except Exception:
        db.session.rollback()
        raise

def save(self):
    db.session.add(self)
    self.commit()
```

The `db.session.add()` is no longer in the try/except but I think it's not a problem because exceptions are raised by the `db.session.commit()` function.

Reply:
In `update` function too [Users model](https://github.com/ovh/celery-director/blob/master/director/models/users.py#L28)